boards/arm/atsamc21n_xpro/atsamc21n_xpro.dts

```diff
@@ -151,3 +152,29 @@
 		};
 	};
 };
+
+&can0 {
+	status = "okay";
+
+	pinctrl-0 = <&can0_default>;
+	pinctrl-names = "default";
+
+	bus-speed = <1000000>;
```

Comment:
```suggestion
	bus-speed = <125000>;
```

Reply:
I don't understand :( i changed it as i receive review from other person.

Have i to change it?

Reply:
Yes please. The only data phase(CAN-FD) should have the higher bit rate of 1MHz.